src/Client.js

```diff
@@ -118,11 +118,12 @@ class Client extends EventEmitter {
                 this.emit(Events.QR_RECEIVED, qr);
             };
             getQrCode();
-            let retryInterval = setInterval(getQrCode, this.options.qrRefreshIntervalMs);
+            this.retryInterval = setInterval(getQrCode, this.options.qrRefreshIntervalMs);
```

Comment:
Looks good! One small thing I would change is make this variable more descriptive since it's now available on client. Please change this throughout the code :)

```suggestion
            this._qrRefreshInterval = setInterval(getQrCode, this.options.qrRefreshIntervalMs);
```

Reply:
Done